driver/registry_default.go

```diff
@@ -256,9 +256,7 @@ func (m *RegistryDefault) RegistrationStrategies() registration.Strategies {
 	if len(m.registrationStrategies) == 0 {
 		for _, strategy := range m.selfServiceStrategies() {
 			if s, ok := strategy.(registration.Strategy); ok {
-				if m.c.SelfServiceStrategy(string(s.ID())).Enabled {
-					m.registrationStrategies = append(m.registrationStrategies, s)
-				}
+				m.registrationStrategies = append(m.registrationStrategies, s)
```

Comment:
The problem here is that these strategies are not only used for handlers but also in other areas (e.g. populating the flow methods). I think this needs to be kept as is!

Reply:
Yeah, I see it. I'll refactor this.

Reply:
Awesome, thanks!